src/lib/y2caasp/widgets/ntp_server.rb

```diff
@@ -81,6 +91,16 @@ def validate
         false
       end
 
+      def help
+        # TRANSLATORS: a help text for the NTP server input field
+        _("<h3>NTP Servers</h3>") +
+          # TRANSLATORS: a help text for the NTP server input field
+          _("<p>Enter the host name or the IP address of the NTP server which will be used for " \
+            "synchronizing the time on this machine</p>") +
+          # TRANSLATORS: a help text for the NTP server input field
+          _("<p>Use comma (,) or space to separate multiple values.</p>")
```

Comment:
can you post screenshot how it looks like? I worry so many paragraph result in too much scrollable text. ( ideally ncurses on 80x24 )

Reply:
![ntp_help](https://user-images.githubusercontent.com/907998/47548892-a6672000-d8fa-11e8-9eef-65f7638f8fc5.png)

Reply:
With some other widget it still fits:
![ntp_help2](https://user-images.githubusercontent.com/907998/47549178-8edc6700-d8fb-11e8-8139-a0e403765d55.png)